assets/setup-wizard/features/feature-description.js

```diff
@@ -1,34 +1,64 @@
-import { __ } from '@wordpress/i18n';
+import { sprintf, __ } from '@wordpress/i18n';
 import { logLink } from '../log-event';
 
+/**
+ * @typedef  {Object} Feature
+ * @property {string} rawTitle Feature title.
+ */
 /**
  * Feature description component
  *
- * @param {Object} props
- * @param {string} props.slug    Feature slug.
- * @param {string} props.excerpt Feature excerpt.
- * @param {string} [props.link]  Feature link.
+ * @param {Object}    props
+ * @param {string}    props.slug               Feature slug.
+ * @param {string}    props.excerpt            Feature excerpt.
+ * @param {string}    [props.link]             Feature link.
+ * @param {Feature[]} [props.selectedFeatures] Feature link.
```

Comment:
Also comment issue.

Reply:
Fixed in the same commit: https://github.com/Automattic/sensei/pull/3297/commits/7ada4c43478f39681d1a9576cd49d7017793d933